scss/bootstrap/overrides/_variables.scss

```diff
@@ -352,10 +347,21 @@ $state-info-text:                $brand-info !default;
 $state-warning-text:             $brand-warning !default;
 $state-danger-text:              $brand-danger !default;
 
-$alert-success-border:           $brand-success !default;
+$alert-success-border:           $btn-success-bg !default;
```

Comment:
wait, why are you using a button-related color (at least, that's what it looks like from the variable name) to define an alert related color? you should define generic color, not colors tied to other components

Reply:
@AkA84 for alert type warning and success, we have slightly different accent then original branding.
Now, in default bootstrap as well, we have this variables overridden as $brand-success while we should have it for this light accent. Should I keep it as this or should I change it adapt value #44CB7E

Also advise for https://github.com/mukeshcompucorp/org.civicrm.shoreditch/blob/fd677954177f9e3bd28b2181e9edd5d325bda6dc/scss/bootstrap/overrides/_variables.scss#L81-L92